contributing.md

```diff
@@ -0,0 +1,16 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via an issue or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+## Pull Request Process
+1. Code review results exist either as draft pull requests or as issues.
+2. Merge requests possible only after code review.
+3. You may merge the Pull Request in once you have the sign-off of two other developers, or if you 
+   do not have permission to do that, you may request the second reviewer to merge it for you.
+## Code of Conduct
```

Comment:
And where is a Code of Conduct? Remove or add.

Reply:
ok